src/index.scss

```diff
@@ -35,6 +34,81 @@ code {
   display: none !important;
 }
 
+// buttons
+.button-blue-fill, .button-blue-outline, .button-red-outline, .button-grey-fill {
+  border-radius: 5px;
+  transition-duration: 0.1s;
+  transition-timing-function: ease-out;
+  &:hover {
+    cursor: pointer;
+  }
+}
+
+.button-blue-fill, .button-grey-fill {
+  border: none;
+}
+
+.button-blue-outline, .button-red-outline {
+  background: white;
+  border-style: solid;
+  border-width: 2px;
+}
+
+.button-blue-fill {
+  color: white;
+  background-color: #54a0f1;
+  &:hover {
+    background-color: #7bb8f9;
+  }
+}
+
+.button-blue-outline {
+  color: #54a0f1;
+  border-color: #54a0f1;
+  &:hover {
+    color: #7bb8f9;
+    border-color: #7bb8f9;
+  }
+}
+
+.button-red-outline {
+  color: #e95f5f;
+  border-color: #e95f5f;
+  &:hover {
+    color: #ee9999;
+    border-color: #ee9999;
+  }
+}
+
+.button-grey-fill {
+  color: #2b2b2b;
+  background-color: #c4c4c4;
+  &:hover {
+    background-color: #747474;
+  }
+}
+
+
+.save-button {
+  margin-bottom: 20px;
+  float: right;
+  padding: 11px 26px;
+  font-weight: 700;
+  font-size: 14px;
+}
+
+.cancel-button {
+  margin-bottom: 20px;
```

Comment:
same as above comment on save-button

Reply:
First example of these buttons that i found just now is in the gallery image upload, and when I remove those margin-bottoms, it makes it look bad. I can't find the edit button on the admin club page to check there, but I imagine if we took away the margin from the buttons, we would need to add it as padding or margin on every container that holds these buttons. Do you want me to try to do that? I'm hoping to merge this PR ASAP before it starts accumulating merge conflicts.

Reply:
Yea good point. For now, I'll just make a ticket that we should come back to this later.